### PR TITLE
Demo reorg

### PR DESCRIPTION
**Pull request by joezuntz**
Reorganize the example scripts and split up CI tests to make results clearer

### CODE REVIEW

File: examples/cosmodc2/pipeline_redmagic_no_shape_noise_full.yml [hunk 1]
```diff
@@ -73,7 +73,7 @@ stages:
     - name: TXTwoPointPlotsFourier
```
Comment (empEvil) [inline examples/cosmodc2/pipeline_redmagic_no_shape_noise_full.yml:74]: TXTwoPointLensCat seems to not exists as a stage

File: examples/2.2i/pipeline_1tract.yml [hunk 2]
```diff
@@ -49,7 +49,7 @@ stages:
     - name: TXPSFDiagnostics
```
Comment (empEvil) [inline examples/2.2i/pipeline_1tract.yml:50]: Still uses TXPhotozStack which doesn't exist any longer

**Review by empEvil — COMMENTED** [review]
I think it is slightly confusing that the examples folder also has the nersc and notebooks folders in them. All the other folders are different iterations of pipelines, but those two folders are not!

File: examples/cosmodc2/pipeline_redmagic_no_shape_noise_full.yml (hunk 1 above)
Comment (joezuntz) [reply examples/cosmodc2/pipeline_redmagic_no_shape_noise_full.yml:74]: fixed

File: examples/2.2i/pipeline_1tract.yml (hunk 2 above)
Comment (joezuntz) [reply examples/2.2i/pipeline_1tract.yml:55]: also fixed!

**Comment by joezuntz** [comment]
I've removed the old NERSC setup command and moved the notebooks folder to the top level.

**Review by empEvil — APPROVED** [review]

**Pull request merged**